cli/cmd/testdata/install_default.golden

```diff
@@ -147,6 +147,7 @@ spec:
         - -tap-addr=127.0.0.1:8088
         - -controller-namespace=conduit
         - -log-level=info
+        - -logtostderr=true
```

Comment:
It is unfortunate that we have to include this on the command line. :( (no change needed for this PR; I hope we can find some more automatic and less noisy solution in the future.)